settings.yml

```diff
@@ -7,6 +7,11 @@ header_categories:
   list_type: category
   description:
     en: A list of categories to render below the header. Defaults to off.
+header_categories_custom_links:
+  type: list
```

Comment:
Is it possible to have a data structure for this? For example to render two fields for each item on the list. I think Discourse does it with one of their native settings, maybe we can check that out and find if it's possible